dev-docs/source/EventWorkspaceDev.rst

```diff
@@ -0,0 +1,93 @@
+.. _Event Workspace Dev:
+
+===========================
+Event Workspace Development
+===========================
+
+.. contents::
+  :local:
+
+The following information will be useful to you if you want to write an
+:ref:`algorithm` that is :ref:`Event Workspace <EventWorkspace>` aware.
+
+Individual Neutron Event Data (TofEvent)
+########################################
+
+The TofEvent class holds information for each neutron detection event
+data:
+
+-  PulseTime: An absolute time of the pulse that generated this neutron.
+   This is saved as an INT64 of the number of nanoseconds since Jan 1,
+   1990; this can be converted to other date and time formats as needed.
+-  tof: Time-of-flight of the neutron, in microseconds, as a double.
+   Note that this field can be converted to other units, e.g. d-spacing.
+
+Lists of Events (EventList)
+###########################
+
+-  The EventList class consists of a list of TofEvent's. The order of
+   this list is not significant, since various algorithms will resort by
+   time of flight or pulse time, as needed.
+
+-  Also contained in the EventList is a std::set of detector ID's. This
```

Comment:
I believe this is a requirement of `Spectrum` which `EventList` inherits from which may be worth mentioning.